app/src/main/java/com/bashkevich/androidfundamentals/movieslist/view/FragmentMoviesList.kt

```diff
@@ -47,10 +48,14 @@ class FragmentMoviesList : Fragment() {
         setUpMoviesListRecyclerView()
         moviesListViewModel.loadMoviesList()
 
-        moviesListViewModel.moviesListLiveData.observe(this.viewLifecycleOwner) { movies ->
+        moviesListViewModel.moviesListLiveData?.observe(this.viewLifecycleOwner) { movies ->
```

Comment:
Why nullable?

Reply:
I want to initialize livedata in viewmodelScope

Reply:
I saw it but suggest to have a solution without null